docs/reference/esql/metadata-fields.asciidoc

```diff
@@ -17,20 +17,24 @@ supported ones are:
   * `_version`: the source document's version. The field is of the type
   <<number,long>>.
 
-To enable the access to these fields, the <<esql-from,`FROM`>> source command needs
-to be provided with a dedicated directive:
+  * <<mapping-source-field,`_source`>>: the original JSON document body 
+  that was passed at index time. The field is loaded as a special `_source` 
+  type. This field is not supported by functions.
```

Comment:
Funnily enough, it _is_ currently supported by `count`, but not `count_distinct`. I opened https://github.com/elastic/elasticsearch/issues/105240

Otherwise, this statement appears to be correct; I'd leave it as-is for now.